### ISSUE TITLE
[CUDA] Instructions do not work

### ISSUE THREAD

**Issue by acxz** [system76/docs#391]
The following line: 
https://github.com/system76/docs/blame/master/_articles/cuda.md#L98

does not work for me.
I get the following error:
`cat: /usr/lib/cuda/version.txt: No such file or directory`

**Comment by sawyermade** [comment]
Same here, system76-cuda-latest literally does nothing. It also doesnt even try to install the damn nvidia driver either. Idk wtf is going on with this, literally tried everything. Cuda is nowhere to be found on the system. Cant even find the directory anywhere or nvcc. I really hate Pop!_OS nowadays, its been trash as of late, a buggy shit show that ends up costing me hours of wasted productivity, I think I will be switching back to Ubuntu here real soon.

**Comment by acxz** [comment]
hmm thats weird, `system76-cuda-latest` works just fine for me. Maybe open another issue for that?

**Comment by sawyermade** [comment]
Okay, I figured out what my problem was. First off, Im using zsh and pop os places the path variables for cuda into /etc/profile.d/ directory, which from my experience only works in terminal login form, not GUI but either way zsh isnt reading from there. I had to manually export the path and lb lib paths, but one thing I didnt notice (when youre reading to fast type thing and see what you wanna see) I thought the path was the standard cuda path in every other distro which is /usr/local/cuda not /usr/lib/cuda. Also, I didnt see any system76 profile stuff to export ld lib path either. Not sure if anyone will run into problems with that or not. Either way, it was a little hacky but my problem seems to stem from using zsh instead of bash.
